pkg/state/helmx.go

```diff
@@ -119,6 +119,22 @@ func (st *HelmState) appendCascadeFlags(flags []string, helm helmexec.Interface,
 	return flags
 }
 
+// append hide-notes flags to helm flags
+func (st *HelmState) appendHideNotesFlags(flags []string, helm helmexec.Interface, ops *SyncOpts) []string {
+	// see https://github.com/helm/helm/releases/tag/v3.16.0
+	if !helm.IsVersionAtLeast("3.16.0") {
+		return flags
+	}
+	switch {
+	// postRenderer arg comes from cmd flag.
+	case ops.HideNotes:
+		flags = append(flags, "--hide-notes")
+	case st.HelmDefaults.HideNotes:
+		flags = append(flags, "--hide-notes")
```

Comment:
Let's remove this and keep it a helmfile command-flag, until we are sure there IS an use-case where you want to toggle it per release.

For now, I consider this as an operation concern that should be toggled at helmfile runtime.